### DISCUSSION TITLE
Data Select.exe (AKA Sonic.exe Characters Select)

### DISCUSSION THREAD

**Discussion by MasterGlitcher2006** [Techokami/SonicWorldsNext#102] [Q&A]
How to make Data Select like in the game Sonic.Exe? Something similar to Not So Simple Sonic Worlds Plus

**Comment by Renhoex** [comment]
The character select screen is pretty bare bones as it is right now.
I would suggest looking up guides on how to set up a generic character select then looking through the script and see what variables get changed for the character.

If it were me I'd just remove all the code referencing the sprite visibility and then add a new sprite for a highlighter that copy's the X position of the currently selected character.

**Comment by DimensionWarped** [comment]
Can you explain what you mean a little more for those of us who haven't played Sonic.exe and/or with that particular rendition of Sonic Worlds?

**Reply by Raptorq1** [reply]
Sonic.exe is a creepy pasta Of the franchise sonic the hedgehog

**Reply by DimensionWarped** [reply]
I know what Sonic.exe is, I don't know what this specific character select screen feature that is wanted is.

**Reply by Raptorq1** [reply]
I think he was asking to do something like this
![Screenshot_20231018-232420](https://github.com/Techokami/SonicWorldsNext/assets/106418027/7cc33701-323f-4853-bc59-b583c7485483)

**Reply by MasterGlitcher2006** [reply]
I meant this:
![7727257-sonicexe-nightmare-beginning-screenshot](https://github.com/Techokami/SonicWorldsNext/assets/148087168/9dfd8c30-35c7-4c9d-839b-72fa830e05fd)

**Reply by Raptorq1** [reply]
Hey if I remember this is exactly the same question to this

https://github.com/Techokami/SonicWorldsNext/discussions/82

oh and I asked reddit for help and well here's the link

https://www.reddit.com/r/godot/comments/16fhfbx/how_would_i_do_this_effect/

**Reply by UndertaleFan6** [reply]
lmao sonic 3 data select but "scary oooOOOoo"
